modules/ocf/manifests/packages/fahclient.pp

```diff
@@ -0,0 +1,18 @@
+class ocf::packages::fahclient {
+  # Folding @ Home
+  ocf::repackage {'fahclient':}
+
+  file {
+    '/etc/fahclient/config.xml':
+      source  => 'puppet:///modules/ocf/packages/fahclient/config.xml',
+      require => Ocf::Repackage['fahclient'],
+      owner   => 'fahclient',
+      mode    => '0664',
+      notify  => Service['FAHClient'];
```

Comment:
Instead of `require`/`notify` here if you prefer you could use arrows for all this:

```puppet
package { 'fahclient':; }
-> file { '/etc/fahclient/config.xml':
  source  => 'puppet:///modules/ocf/packages/fahclient/config.xml',
  owner   => 'fahclient',
  mode    => '0664',
}
~> service { 'FAHClient':; }
```

Some people prefer them, some don't, idk really